app/src/renderer/components/monitor/PageBlockchain.vue

```diff
@@ -1,25 +1,18 @@
 <template lang="pug">
-page(title='Blockchain')
+page(title='Monitor')
   div(slot="menu"): tool-bar
-    a(@click='setSearch(true)')
-      i.material-icons search
-      .label Search
-
   template(v-if="blockchain")
-    part(title='Metadata')
-      list-item(dt='Network Name' :dd='status.node_info.network')
-      list-item(dt='App Version' :dd='version')
-      list-item(dt='Tendermint Version' :dd='status.node_info.version')
+    part(title='Versions')
```

Comment:
i'd like to remove this section and turn this page into a block explorer

Reply:
I am pro keeping the versions in there. Could be helpful for developers (us) to give an easy entry point for submitting errors. BUT: if we don't have the versions yet, let's add it as an idea and do it later.

Reply:
We also get a whole bunch of versions for the connected node via `node_info.other` in `gaia client rpc status`.